src/StopWatch.cpp

```diff
@@ -18,4 +19,17 @@ void StopWatch::tok() {
 void StopWatch::print_stats() {
   auto duration = std::chrono::duration_cast<std::chrono::milliseconds>(stop-start).count();
   std::cout << name <<  ": Execution Time: " << duration << " ms" << std::endl;
+  measurements.push_back(duration);
```

Comment:
wouldn't it make more sense to append to measurements in the tok() call?

Reply:
ja, ist angepasst